queries.sql

```diff
@@ -1,3 +1,58 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT 
+* 
+from 
+animals 
+WHERE 
+name LIKE '%mon';
+
+SELECT
+name
+FROM 
+animals
+ WHERE
+ date_of_birth BETWEEN 2016 AND 2019;
```

Comment:
- Good job done so far, but according to the project requirements, you are to attach the screenshots for all queries, I cannot find the screenshot for this query. Please fix this 🙏